CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.14.0 (2020-12-01)
```

Comment:
Should this be updated to `2020-12-02`? I'm assuming this won't go out until then.

Reply:
Done.